src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,37 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.task.Task;
+import duke.util.TaskList;
+import duke.util.Storage;
+import duke.util.Ui;
+
+/**
+ * Represents a command to add a task to the task list.
+ */
+public class AddCommand extends Command {
+    String input;
```

Comment:
Missing access modifier here!
```suggestion
    private String input;
```